compiler/src/iree/compiler/GlobalOptimization/Passes.h

```diff
@@ -54,6 +54,10 @@ std::unique_ptr<Pass> createDetachElementwiseFromNamedOpsPass();
 std::unique_ptr<OperationPass<mlir::ModuleOp>>
 createEraseUnusedLinalgOperands();
 
+// Expands vectors in vector/matrix operations into matrices to allow for
+// tiling.
+std::unique_ptr<Pass> createExpandVectorsPass();
+
```

Comment:
The pass just rewrites the ops in favor of matmuls. `for tiling` is not part of the pass. We do not add one of potential purposes to comments. Can you drop it?
```suggestion
// Expands vectors in vector/matrix operations into linalg.batch_matmul/matmul forms.
std::unique_ptr<Pass> createExpandVectorsPass();

```